hoomd/hpmc/ComputeSDF.h

```diff
@@ -149,14 +149,17 @@ template<class Shape> class ComputeSDF : public Compute
 
     //! Return an sdf
     virtual pybind11::array_t<double> getSDF();
+    virtual pybind11::array_t<double> getSDF_expansion();
 
     protected:
     std::shared_ptr<IntegratorHPMCMono<Shape>> m_mc; //!< The parent integrator
     double m_xmax;                                   //!< Maximum lambda value
     double m_dx;                                     //!< Histogram step size
 
-    std::vector<double> m_hist; //!< Raw histogram data
-    std::vector<double> m_sdf;  //!< Computed SDF
+    std::vector<double> m_hist;           //!< Raw histogram data
+    std::vector<double> m_sdf;            //!< Computed SDF
```

Comment:
Will need to name these and related variables with `_compression` to avoid ambiguity.